README.md

```diff
@@ -27,7 +27,7 @@ For more information, refer to our [documentation](https://docs.adyen.com/) or t
 ## Prerequisites
 - [Adyen test account](https://docs.adyen.com/get-started-with-adyen)
 - [API key](https://docs.adyen.com/development-resources/api-credentials#generate-api-key). For testing, your API credential needs to have the [API PCI Payments role](https://docs.adyen.com/development-resources/api-credentials#roles).
-- Ruby >= 2.1
+- Ruby >= 2.5
```

Comment:
Will this result in a new major release?

Reply:
A quick google tells me that ruby 2.x is EOL. Should we upgrade the Lib to support >v3.0.x?

Reply:
Let's keep 2.7 for now as it has stopped being supported only a month ago.